src/main/java/frc/robot/SwerveModule.java

```diff
@@ -0,0 +1,119 @@
+package frc.robot;
+
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModulePosition;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+
+import frc.lib.math.Conversions;
+import frc.lib.util.CTREModuleState;
+import frc.lib.util.SwerveModuleConstants;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.DemandType;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.ctre.phoenix.sensors.CANCoder;
+
+public class SwerveModule {
```

Comment:
This is only used by the Swerve subsystem. Move to `subsystems`? Feels oddly placed here.

Reply:
Moved to its own folder with other swerve stuff in `subsystems/swerve`